src/api/axiosInstance.ts

```diff
@@ -1,9 +1,30 @@
 import axios from 'axios'
+import { useNavigate } from 'react-router'
 
 const axiosInstance = axios.create({
-  baseURL: 'http://13.125.102.156:8080',
+  baseURL: 'https://home-try.13.125.102.156.sslip.io',
```

Comment:
baseURL처럼 변경될 수 있는 값은 코드를 변경하기 보다는 환경변수를 주입하고 코드 변경없이 이 값만 수정할 수 있게 구성할 수 있습니다. 완료가 다가오는 지금 시점에 꼭 수정을 해야하는건 아니구요, 시간되시면 읽어보시고 이런게 있구나 정도로 넘어가도 충분할 거 같습니다.


```suggestion
  baseURL: process.env.REACT_APP_API_URL,
```

- [Adding Development Environment Variables In `.env`](https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env)